src/Network/Network/PrivateLinkService/PrivateLinkServiceProvider/ProviderConfiguration.cs

```diff
@@ -47,6 +47,7 @@ static ProviderConfiguration()
             RegisterConfiguration("Microsoft.Synapse/workspaces", "2019-06-01-preview");
             RegisterConfiguration("Microsoft.Web/sites", "2019-08-01");
             RegisterConfiguration("Microsoft.Web/hostingEnvironments", "2020-10-01");
+            RegisterConfiguration("Microsoft.DigitalTwins/digitalTwinsInstances", "2020-12-01");
```

Comment:
Please insert the code to an appropriate position so that the list of private link is in lexicographical order.

Reply:
Changed code to an appropriate position so that the list of private link is in lexicographical order